docker/node1.env

```diff
@@ -10,4 +10,6 @@ SEBAK_VALIDATORS=self https://127.0.0.1:2822?address=GAYGELM74WJMKSLDN5YP2VAMP64
 # This node's public key
 SEBAK_GENESIS_BLOCK=GDIRF4UWPACXPPI4GW7CMTACTCNDIKJEHZK44RITZB4TD3YUM6CCVNGJ
 # common account
-SEBAK_COMMON_ACCOUNT=GDIV7E7WMBHCYYZHBV6MXHCHC4VCLU4BTCEM2YUQK73EKIM5TPC66VYR
+SEBAK_COMMON_ACCOUNT=GCTVCU764UPXKRJW5DS5AWF5ETCCIYPZTWXN5U5CLUNAAJH6D5NGEIIH
```

Comment:
I couldn't find the secret key for the common account.
Hence, I created a new one.

Reply:
@Charleslee522 FYI.